hieradata/cluster/ruka.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 classes:
   - "profile::core::ifdown"
+  - "profile::core::velero"
```

Comment:
Is the intent to only install velero on ruka and vk8s?  If not, this should be part of the rke role.

Reply:
Is what this branch contemplates. The integration of all others is separated into several branches. I recalling you saying "small changes at the time"